test/OpenTelemetry.Instrumentation.SqlClient.Tests/SqlActivitySourceHelperTests.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using OpenTelemetry.Instrumentation.SqlClient.Implementation;
+using Xunit;
+
+namespace OpenTelemetry.Instrumentation.SqlClient.Tests;
+
+public class SqlActivitySourceHelperTests
+{
+    [Fact]
+    public void ShouldCalculateDuration()
```

Comment:
Failing test on ubuntu

```text
  Failed OpenTelemetry.Instrumentation.SqlClient.Tests.SqlActivitySourceHelperTests.ShouldCalculateDuration [24 ms]
  Error Message:
   Assert.Equal() Failure: Values differ
Expected: 10
Actual:   0.10000000000000001
  Stack Trace:
     at OpenTelemetry.Instrumentation.SqlClient.Tests.SqlActivitySourceHelperTests.ShouldCalculateDuration() in /home/runner/work/opentelemetry-dotnet-contrib/opentelemetry-dotnet-contrib/test/OpenTelemetry.Instrumentation.SqlClient.Tests/SqlActivitySourceHelperTests.cs:line 16
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)
```

Reply:
Dropped the test for now, was just a nice bonus but not that valuable.